modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1324,13 +1327,39 @@ CV__DNN_INLINE_NS_BEGIN
           */
          CV_WRAP ClassificationModel(const Net& network);
 
+         /**
+          * @brief The default for SoftMax option is false.
+          * If this option is true, SoftMax is applied after forward inference within the classify() function
```

Comment:
Please add "empty" line after `@brief` statement.

http://pullrequest.opencv.org/buildbot/export/pr/21692/docs/d0/dd5/classcv_1_1dnn_1_1ClassificationModel.html

Reply:
Got it!